dtmi/azureiot/paad/iotphone-1.json

```diff
@@ -0,0 +1,106 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:azureiot:PaaD:IoTPhone;1",
+  "@type": "Interface",
+  "displayName": "IoT Phone",
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "AppTheme",
+      "writable": true,
+      "schema": {
+        "@type": "Enum",
+        "valueSchema": "string",
+        "enumValues": [
+          {
+            "name": "light",
+            "enumValue": "Light"
+          },
+          {
+            "enumValue": "Dark",
+            "name": "dark"
+          }
+        ]
+      }
+    },
+    {
+      "@type": "Command",
+      "name": "restartApp"
```

Comment:
Is it ok to have commands without request/response?

Reply:
> Is it ok to have commands without request/response?

yes